classes/app.rb

```diff
@@ -0,0 +1,136 @@
+require_relative 'person'
+require_relative 'decorator'
+require_relative 'book'
+require_relative 'rental'
+require_relative 'teacher'
+require_relative 'student'
+require_relative 'classroom'
+
+class App
+  def initialize
+    @books = []
+    @people = []
+  end
+
+  def list_all_books
+    if @books.empty?
+      puts 'There are no books in the library'
+    else
+      @books.each do |book|
+        puts "Title: #{book.title}, Author: #{book.author}".capitalize
+      end
+    end
+  end
+
+  def list_people
+    if @people.empty?
+      puts 'There are no people in the library'
+    else
+      @people.each do |person|
+        puts "[#{person.class}] Name: #{person.name}, Age: #{person.age}, ID: #{person.id}"
+      end
+    end
+  end
+
+  def create_person
+    person_type = person_option
+    return if person_type.nil?
+
+    print 'Name: '
+    name = gets.chomp
+    print 'Age: '
+    age = gets.chomp
+
+    case person_type
+    when '1'
+      create_student(name, age)
+    when '2'
+      create_teacher(name, age)
+    end
+  end
```

Comment:
- [ ] You did a great job working on this milestone! 🚀 However, there's still room to improve! Currently, I'm having an error trying to create a new person either a student or a teacher:

![image](https://user-images.githubusercontent.com/77069011/233805821-f0b281b9-7a45-4473-a069-d0e4f381f703.png)

💡 Hint: Refer to where you're calling `super()` on each of these classes `Student` and `Teacher` Pay special attention to how you're passing the `parent_permission` attribute vs how your `Person` class expects to receive it (Refer to named parameters)! 🤓 

Please consider fixing this! 🔧